.gitignore

```diff
@@ -17,3 +17,6 @@
 
 # Ignore secrets
 .env
+
+# Ignore rbenv ruby version
+.ruby-version
```

Comment:
I prefer to leave this checked in. Your Gemfile will already demand 2.2.2, and having this in the repo just makes everyone's lives easier.